VocaDb.ResXFileCodeGenerator/FileOptions.cs

```diff
@@ -0,0 +1,59 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace VocaDb.ResXFileCodeGenerator;
+
+internal record FileOptions
```

Comment:
Please apply the `sealed` modifier.

Reply:
old java dev? It doesn't matter in C# since the default is non-virtual. I have applied it nonetheless.

Reply:
It's true that I used to use Java many years ago, but I tend to mark classes as sealed as much as possible unless they really need to be inherited (I think classes are sealed by default in newer languages). It's relatively easy to remove the sealed modifier, but hard to add it afterwards (if they have some derived classes already). But it's not a big problem anyway as long as the class is marked as internal.